microceph/api/osds.go

```diff
@@ -0,0 +1,51 @@
+package api
```

Comment:
**suggestion (non-blocking):** 

It will be nice if you can separate the API part to another PR, which may be more easy for reviewer.